hyperspy/drawing/_widgets/rectangles.py

```diff
@@ -40,6 +40,10 @@ class SquareWidget(Widget2DBase):
     def __init__(self, axes_manager, **kwargs):
         super(SquareWidget, self).__init__(axes_manager, **kwargs)
 
+    def connect(self, ax):
+        super(SquareWidget, self).connect(ax)
+        canvas = ax.figure.canvas
```

Comment:
Some leftover, which needs to be removed!
```suggestion
    def connect(self, ax):
        super(SquareWidget, self).connect(ax)
        canvas = ax.figure.canvas
```